x/protocolpool/keeper/msg_server.go

```diff
@@ -201,6 +201,20 @@ func (k MsgServer) CancelContinuousFund(ctx context.Context, msg *types.MsgCance
 		return nil, fmt.Errorf("failed to remove continuous fund for recipient %s: %w", msg.RecipientAddress, err)
 	}
 
+	if err := k.RecipientFundPercentage.Remove(ctx, recipient); err != nil {
+		return nil, fmt.Errorf("failed to remove recipient fund percentage for recipient %s: %w", msg.RecipientAddress, err)
+	}
+	// clean up state after canceling continuous fund, all funds must be removed before
+	value, err := k.RecipientFundDistribution.Get(ctx, recipient)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get recipient fund distribution for recipient %s: %w", msg.RecipientAddress, err)
+	}
+	if value.IsZero() {
```

Comment:
Can this be non zero?  The amount is cleared on `withdrawRecipientFunds` before.